bin/express

```diff
@@ -312,10 +319,27 @@ function createApplication(app_name, path) {
   });
 }
 
+function addViewEngine(app, viewEngine) {
+  if (program.view == 'none') {
+    app = app.replace('{view-engine-set-up}', '');
+    return app.replace('{render-error}', `res.sendFile('error.html', {
+    root: __dirname + '/public/'
+  });`);
+  } else {
+    app = app.replace('{view-engine-set-up}', `
+// view engine setup
```

Comment:
Can the indenting get fixed here? Looks really weird with this offset indenting.